encode_utils/connection.py

```diff
@@ -1212,13 +1212,13 @@ def remove_props(self, rec_id, props=[]):
             prop = profile.get_property_from_name(prop_name)
             if prop.is_required:
                 raise Exception("Can't remove required property")
-            elif prop.is_not_submittable:
+            elif prop.is_not_submittable():
```

Comment:
Should change `is_not_submittable` to be an `@property`, then this line will work as written. It is used like this in another place, which is also broken.

https://github.com/StanfordBioinformatics/encode_utils/blob/4cde56ff9418177bf4297c4ec0b6695f0e4eb623/encode_utils/profiles.py#L189